src/pages/public_registration/PublicFilesContent.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import CloudDownloadIcon from '@material-ui/icons/CloudDownload';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import LockIcon from '@material-ui/icons/Lock';
+import { File, licenses } from '../../types/file.types';
+import { downloadFile } from '../../api/fileApi';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+import ButtonWithProgress from '../../components/ButtonWithProgress';
+import {
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@material-ui/core';
+import { PublicRegistrationContentProps } from './PublicRegistrationContent';
+
+const StyledTableCell = styled(TableCell)`
+  font-size: 1rem;
+  word-wrap: break-word;
+`;
+
+const StyledNameTableCell = styled(StyledTableCell)`
+  min-width: 10rem;
+  font-weight: 700;
+  line-break: anywhere;
+`;
+
+const StyledLicenseImg = styled.img`
+  cursor: pointer;
+`;
+
+const StyledFilesContent = styled.div``;
+
+const PublicFilesContent = ({ registration }: PublicRegistrationContentProps) => {
+  const { t } = useTranslation('common');
+
+  const publiclyAvailableFiles = registration.fileSet.files.filter((file) => !file.administrativeAgreement);
+
+  return (
+    <StyledFilesContent>
+      <Typography variant="h4" component="h2" gutterBottom>
+        {t('registration:files_and_license.files')}
+      </Typography>
+
+      <TableContainer>
```

Comment:
Table virker egentlig bare sånn passe på mobil siden det vil bli en horisontal scroll.  men det ser ok ut nå, så tenker vi kan si det er greit nok foreløpig og ha den til senere

Reply:
Jeg tenker horisontal scroll er helt innafor på mobil! Blir mer jobb enn belønning å skulle omorganisere alt kun for mobil (som er tredjepri)

Reply:
I skissene var det egentlig tegnet som Card, men tok meg litt kunstnerisk frihet og la det inn i Table slik at vi fikk heading per kolonne også. Hvis vi hadde brukt Card med grid i stedet kunne vi sikkert fått det bra på både desktop og mobil